.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -35,10 +35,13 @@ Steps to reproduce the behavior:
 
 ### Setup
 <!--
-1. ROOT version
-2. Operating system
-3. How you obtained ROOT, such as `dnf install` / binary download / you built it yourself.
+Unix: `root -b -q | xclip -sel clip`
```

Comment:
What's the point for also showing this when ".gh" does the job now? Isn't the last line sufficient?

Reply:
I wanted to leave this for backward compatibility. Most people won't have latest master installed until a year or so.

Reply:
Or maybe we can state the recommended way depending on the version?

Reply:
I like the idea of a version specific text!